gui.js

```diff
@@ -3027,11 +3027,10 @@ IDE_Morph.prototype.importMedia = function (folderName) {
     // costumes, backgrounds or sounds
     var myself = this,
         msg = this.showMessage('Opening ' + folderName + '...');
-    this.getMediaList(
-        folderName,
-        function (items) {
-            msg.destroy();
-            myself.popupMediaImportDialog(folderName, items);
+    myself.getURL(
+        folderName + "/catalog.json",
+        function (txt) {
+            myself.popupMediaImportDialog(folderName, JSON.parse(txt));
```

Comment:
I'll need to this about this - we have a getMediaList function or something similar.